website/notifications/emails.py

```diff
@@ -60,10 +60,14 @@ def notify_mentions(event, user, node, timestamp, **context):
 def notify_global_event(event, sender_user, node, timestamp, recipients, template=None, context=None):
     event_type = utils.find_subscription_type(event)
     sent_users = []
+    if not context:
+        context = {}
 
     for recipient in recipients:
         subscriptions = get_user_subscriptions(recipient, event_type)
         context['is_creator'] = recipient == node.creator
+        if node.provider:
+            context['has_psyarxiv_chronos_text'] = node.has_permission(recipient, ADMIN) and 'psyarxiv' in node.provider.name.lower()
```

Comment:
this probably shouldn't be based on provider `name`, and definitely shouldn't use a hard-coded `'psyarxiv'` string -- is there no way to tell whether a given provider has chronos enabled? seems like would belong as a boolean field on the provider

Reply:
>  is there no way to tell whether a given provider has chronos enabled? 

None that I can see!

Reply:
yeah there's not -- approving because we decided to do that later